source/module_io/write_wfc_r.cpp

```diff
@@ -4,21 +4,20 @@
 // DATE :   2021-11-21
 //======================
 
+#include "module_base/timer.h"
+#include "module_base/tool_title.h"
+#include "module_hamilt_pw/hamilt_pwdft/global.h"
 #include "write_wfc_r.h"
 
 #include <cstdlib>
 #include <fstream>
 #include <stdexcept>
 
-#include "module_base/timer.h"
-#include "module_base/tool_title.h"
-#include "module_hamilt_pw/hamilt_pwdft/global.h"
-
 namespace ModuleIO
 {
-	// write ||wfc_r|| for all k-points and all bands
-	// Input: wfc_g(ik, ib, ig)
-	// loop order is for(z){for(y){for(x)}}
+// write ||wfc_r|| for all k-points and all bands
+// Input: wfc_g(ik, ib, ig)
+// loop order is for(z){for(y){for(x)}}
```

Comment:
I am happy to see you delete those strange tab(s) and indents, but for "loop order is ...", there is a clear standard for cube file here: http://www.paulbourke.net/dataformats/cube/
you can replace this annotation with this link.